spec/fixtures/merge_request_dependencies.json

```diff
@@ -0,0 +1 @@
+[{"id":1234,"blocking_merge_request":{"id":2,"target_branch":"master","source_branch":"api2","project_id":3,"title":"New feature 2","closed":false,"merged":false,"author":{"id":1,"email":"john@example.com","name":"John Smith","blocked":false,"created_at":"2012-10-19T05:56:05Z"},"assignee":{"id":2,"email":"jack@example.com","name":"Jack Smith","blocked":false,"created_at":"2012-10-19T05:56:14Z"}},"blocked_merge_request":{"id":1,"target_branch":"master","source_branch":"api","project_id":3,"title":"New feature","closed":false,"merged":false,"author":{"id":1,"email":"john@example.com","name":"John Smith","blocked":false,"created_at":"2012-10-19T05:56:05Z"},"assignee":{"id":2,"email":"jack@example.com","name":"Jack Smith","blocked":false,"created_at":"2012-10-19T05:56:14Z"}}}]
```

Comment:
**thought (non-blocking):** Is there any reason to have the fixtures minimized, and not pretty-formatted? I would find them easier to maintain.

Reply:
Mostly to reduce file sizes. They're rarely maintained after being added, and I don't see a problem with prettifying and minimizing them again in any modern editor.